client/src/Auth/Login.js

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useState } from "react";
+import { Link, useLocation } from "react-router-dom";
+
+import {useNavigate} from 'react-router-dom';
+import Spinner from '../utils/Spinner';
+import {useAuth} from '../contexts/AuthContext';
+
+import axios from 'axios';
+
+import "./Auth.css";
+
+const Login = () => {
+
+  useEffect(() => {
+    window.scroll({ behavior:'smooth',top:0 });
+  },[])
+
+    const [state,setState] = useState({email:"",password:""});
```

Comment:
Suggestion: you can use a reducer here.

Reply:
No we don't need reducer here this state is just for email,password field why use reducer? and make it complex